### ISSUE TITLE
Peaklist XML import gives null pointer exception

### ISSUE THREAD

**Issue by photocyte** [mzmine/mzmine2#883]
### Basic information

* My operating system: MacOS / Linux
* My MZmine version:  2.53

### What happened

Hi there, 
If you export an XML file from a peaklist, it seems that XML cannot be reimported without causing a null pointer error. Quick tests show it is not caused by cropped vs original raw files (error happens with both), or the original peaklist still being present. Ran into this issue on both MacOS and Linux.  Example toy MZmine project + XML file to reproduce issue in attached .zip file.

<img width="264" alt="Screen Shot 2020-12-09 at 2 11 58 PM" src="https://user-images.githubusercontent.com/12517705/101695740-e67db400-3a29-11eb-894c-0ab71cbb58fe.png">

[XML_issue.zip](https://github.com/mzmine/mzmine2/files/5669088/XML_issue.zip)

**Comment by tomas-pluskal** [comment]
Hi Tim,
sadly the XML export/import modules were never maintained, so I am not surprised they don't work anymore. I think it makes more sense to use mzTab-M as an output format from MZmine.

**Comment by photocyte** [comment]
I agree with the standardized export formats (mzTab or other), however, something useful from the peaklist that I believe is missing from the tab delimited formats is the isotopic pattern, and, the rt,intensity profile of the ion chromatogram associated with the peak. For what it's worth, the exported XML seems properly formed / not obviously broken.  I'll consider troubleshooting MZmine2 with breakpoints to debug this issue, but would be nice if there could be a MZmine2 release if the bug is fixed :)

**Comment by ckbarlow79** [comment]
Hi Tim,
It looks like I've run into the same issue as you.  Just wondering did you ever maker any progress on the xml import problem?
Cheers,
Chris

**Comment by photocyte** [comment]
Hi there Chris, in fact I did. See here: #884 But, MZmine2 is under a feature freeze in order to focus work on MZmine3 - it's unlikely there will be another MZmine2 release. You can try to compile MZmine2 yourself to incorporate that fix. There is documentation around the MZmine website to configure an IDE (I use Eclipse) to compile MZmine2. E.g. here http://mzmine.github.io/eclipse_tutorial.html , but search other places too. I seem to remember that the change to the new build system is better described in an issue somewhere.

P.S. to MZmine developers to consider including a MZmine3 export/import format which includes the rt,intensity profile for the ion chromatogram.

**Comment by ckbarlow79** [comment]
Hi Tim,
Thanks for the detailed response, I will check it out.
